server/services/cohorts-lookup.js

```diff
@@ -0,0 +1,15 @@
+// @flow
+import Prismic from 'prismic.io';
+
+export const getCohorts = async () => {
+  const api = await Prismic.api('http://wellcomecollection.prismic.io/api');
+  const response = await api.query(Prismic.Predicates.at('document.type', 'featurescohort'));
```

Comment:
As a point, I am really not happy that we have to use two async calls here, the latency introduced would be too much from my point of view.

Given that this runs as a task however, makes me less worried.